package-canlib.bash

```diff
@@ -1,56 +1,58 @@
 #!/bin/bash
-
-# Exit if any command fails
 set -e
 
-PWD=$(pwd)
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 OS_VER=$(lsb_release -cs)
+SDK_COMMIT="$1"
+VER_SUFFIX="$2"
+
+if [ -z "$SDK_COMMIT" ]; then
+  SDK_COMMIT=master
+fi
 
 echo ""
 echo "Remember to modify the changelog in debian-canlib to include the current release notes."
 echo "If you need to modify it now, hit [CTRL+C]. Otherwise hit [ENTER] to continue."
 read STUFF
 
+cd "$SCRIPT_DIR"
+
 # Check for required files/folders
-if [ ! -e "$PWD/canlib/debian-canlib" ]; then
+if [ ! -e "canlib/debian-canlib" ]; then
   echo ""
   echo "debian-canlib directory not found in canlib folder. Exiting..." 1>&2
   exit -1
 fi
 
-if [ ! -e "$PWD/canlib/Makefile-canlib" ]; then
+if [ ! -e "canlib/Makefile-canlib" ]; then
   echo ""
   echo "Makefile-canlib not found in canlib folder. Exiting..." 1>&2
   exit -1
 fi
 
-# Delete BUILD directory if it exists
-if [ -d "$PWD/BUILD" ]; then
-  rm -r BUILD/
-fi
-
+rm -rf BUILD/
 mkdir BUILD
 cd BUILD/
 
 # Clone linuxcan folder
-git clone https://github.com/astuff/kvaser-linuxcan
+git clone --branch "$SDK_COMMIT" https://github.com/astuff/kvaser-linuxcan
 
 mv kvaser-linuxcan/ kvaser-canlib/
 
 # Get version of linuxcan
 VERSION=$(cat kvaser-canlib/moduleinfo.txt | grep version | sed -e "s/version=//" -e "s/_/./g" -e "s/\r//g")
 DEBIAN_VERSION=${VERSION}-0ubuntu0~ppa
 
-if [ $# -gt 0 ]; then
-  DEBIAN_VERSION=${DEBIAN_VERSION}$1
+if [ -n "$VER_SUFFIX" ]; then
+  DEBIAN_VERSION=${DEBIAN_VERSION}$VER_SUFFIX
 else
   DEBIAN_VERSION=${DEBIAN_VERSION}0
 fi
```

Comment:
If `$VER_SUFFIX` is empty, adding as a suffix won't hurt anything so you can do this without the `if`.

Reply:
I thought it would be best to leave the existing functionality so it always defaults to `0`, then later if you need to release a `1` version, it will already follow the order.

Reply:
Sure - makes sense.